hw1/task2/index.js

```diff
@@ -0,0 +1,35 @@
+// https://github.com/Keyang/node-csvtojson
+
+import fs from "fs";
+import csv from "csvtojson";
+import { pipeline } from "stream";
+
+const csvContent = "./csv/";
+
+fs.readdir(csvContent, (err, files) => {
+  if (err) {
+    console.error("Pipeline failed.", err);
+    throw Error;
+  }
+
+  files.forEach((file) => {
+    if (file && file.length) {
+      const fileName = file.split(".");
+      const fileType = fileName.pop();
+
+      fileType === "csv" &&
+        pipeline(
+          fs.createReadStream(`./csv/${file}`),
+          csv(),
```

Comment:
It would be better if you added your transform stream

Reply:
As we've discussed I'll investigate this later.